### PR TITLE
Finished 02_exercise

### PR DESCRIPTION
**Pull request by DonVerba**
Testing pull request. I added 02_exercise.

### CODE REVIEW

File: 02_exercise/src/main.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#include <stdlib.h>
+
+int factorial(int n)
+{
+    int i = 1;
+    while (n>1)
+    {
+        i = i*n;
+        //  Here, we are creating a new local variable that hides previous n.
+        // int n = n-1;
```
Comment (rafa-martin) [inline 02_exercise/src/main.c:11]: Nice solve dude

File: 02_exercise/src/main.c [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    return i;
+}
+
+int main()
+{
+    int fac4 = factorial(4);
+    int fac5 = factorial(5);
+
+    printf("4! = %d, 5! = %d\n", fac4, fac5);
+    return EXIT_SUCCESS;
```
Comment (rafa-martin) [inline 02_exercise/src/main.c]: This isnt in oficial problem

File: 02_exercise/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Exercise 1
```
Comment (rafa-martin) [inline 02_exercise/README.md]: Change to correct number

**Review by rafa-martin — CHANGES_REQUESTED** [review]
You only need apply small changes. Good job.

File: 02_exercise/src/main.c (hunk 1 above)
Comment (DonVerba) [reply 02_exercise/src/main.c:11]: Thanks

File: 02_exercise/src/main.c [hunk 4]
```diff
@@ -20,5 +20,5 @@ int main()
     int fac5 = factorial(5);
 
     printf("4! = %d, 5! = %d\n", fac4, fac5);
-    return EXIT_SUCCESS;
+    return 0;
```
Comment (rafa-martin) [inline 02_exercise/src/main.c:23]: Ok, that ok.

**Review by rafa-martin — COMMENTED** [review]
That all??

File: 02_exercise/README.md (hunk 3 above)
Comment (DonVerba) [reply 02_exercise/README.md]: I also change the number and description

Comment (rafa-martin) [reply 02_exercise/README.md]: Where??

Comment (DonVerba) [reply 02_exercise/README.md]: Next comit #2

File: 02_exercise/README.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
-# Exercise 1
-This exercise consists in write a program that calculate the first 100 triangular numbers.
+# Exercise 2
```
Comment (DonVerba) [inline 02_exercise/README.md:1]: is there the changes requested in #1

Comment (rafa-martin) [reply 02_exercise/README.md:1]: Ahh, ok. Thats all. Thanks a lot :D

**Pull request merged**